documentation/dotnet-trace-instructions.md

```diff
@@ -0,0 +1,132 @@
+# Trace for performance analysis utility (dotnet-trace)
+
+The dotnet-trace is a cross-platform CLI global tool that enables the collection of .NET Core traces of a running process without any native profiler involved. It is built around the EventPipe technology of the .NET Core runtime as an alternative to ETW on Windows and LTTNG on non-Windows.
+
+## Installing dotnet-trace
+
+The first step is to install the dotnet-trace CLI global tool. This requires the 2.1 .NET Core SDK to be installed.
+
+<!-- TODO: Make package globally available. -->
+```cmd
+$ dotnet tool install --global dotnet-trace --version 1.0.3-preview5.19217.3 --add-source https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
+You can invoke the tool using the following command: dotnet-trace
+Tool 'dotnet-trace' (version '1.0.3-preview5.19217.3') was successfully installed.
+```
+
+## Using dotnet-trace
+
+In order to collect traces you will need to:
+
+- First, find out the process identifier (pid) of the .NET Core 3.0 app to collect traces from
+
+  - On Windows, there are options such as using the task manager or the `tasklist` command on the cmd window.
+  - On Linux, the trivial option could be using `pidof` on the terminal window.
+
+- Then, run the following command:
+
+```cmd
+dotnet-trace collect --process-id <PID> --providers Microsoft-Windows-DotNETRuntime:0x00000004C14FCCBD:4
```

Comment:
What does this filter turn on?

Reply:
Isn't this the default runtime provider keyword PerfView uses? Maybe we can just skip the keyword and level on the instruction :-)